### PR TITLE
Fix for Cannot convert properties containing value and tree on same l…

### PR DESCRIPTION
**Pull request by mariuszs**
…evel #7

### CODE REVIEW

File: library/src/test/groovy/io/codearte/props2yaml/PropertiesFromWikipediaSpec.groovy [hunk 1]
```diff
@@ -21,6 +21,15 @@ class PropertiesFromWikipediaSpec extends Specification {
             yaml !=~ 'exclamation'
     }
 
+    def "Should read wikipedia log4j properties"() {
```
Comment (dst) [inline library/src/test/groovy/io/codearte/props2yaml/PropertiesFromWikipediaSpec.groovy]: wikipedia?

File: library/src/test/groovy/io/codearte/props2yaml/ConverterSpec.groovy [hunk 2]
```diff
@@ -50,4 +50,16 @@ line2'''
             yaml =~ "a: b"
     }
 
+    def "Should convert nested elements"() {
+        given:
+            String props = 'a = x\n' +
+                    'a.b = y'
+        when:
+            String yaml = new Props2YAML(props).convert();
+            println yaml
```
Comment (dst) [inline library/src/test/groovy/io/codearte/props2yaml/ConverterSpec.groovy]: I'd remove println

File: library/src/test/groovy/io/codearte/props2yaml/PropertiesFromWikipediaSpec.groovy (hunk 1 above)
Comment (mariuszs) [inline library/src/test/groovy/io/codearte/props2yaml/PropertiesFromWikipediaSpec.groovy]: thanks :)

**Comment by jkubrynski** [comment]
:+1:

**Pull request merged**